syntax = "proto3";

service GrpcServiceBenchmark {
  rpc get_answer (AnswerRequest) returns (AnswerReply) {}
  rpc get_blob (EmptyRequest) returns (BlobResponse) {}
}

message AnswerRequest {
  int32 number = 1;
}

message AnswerReply {
  int32 result = 1;
}

message EmptyRequest { }

message BlobResponse {
  bytes data = 1;
}
